mappings/net/minecraft/client/render/WorldRenderer.mapping

```diff
@@ -405,6 +419,7 @@ CLASS net/minecraft/unmapped/C_sfkkabhx net/minecraft/client/render/WorldRendere
 		ARG 4 y
 		ARG 6 z
 		ARG 8 color
+	METHOD m_znnjtopn createChunkBuilderDebugLine ()Ljava/lang/String;
```

Comment:
```suggestion
	METHOD m_znnjtopn getChunkBuilderDebugString ()Ljava/lang/String;
```
I know I said the previous thing, but I want to match the other functions like this.